### ISSUE TITLE
yt.load_sample() has issues loading datasets with null "load_name" in the sample_data_registry

### ISSUE THREAD

**Issue by munkm** [yt-project/yt#2699]
### Bug report

**Bug summary**

When going using the load_sample function with yt I am able to download a sample dataest, but if the "load_name" val is null load_sample doesn't work. We should make the behavior more intutitive or supply a default file for affected datasets in the sample_data_registry.json . I've tried this with a few datasets that have null for this keyword and all have the same issue. 

**Code for reproduction**

```python
import yt
ds = yt.load_sample("Orbit")
```

**Actual outcome**
```
yt : [WARNING  ] 2020-06-26 13:20:52,112 tqdm is not installed, progress bar can not be displayed.
yt : [INFO     ] 2020-06-26 13:20:52,140 Files located at /Users/madicken/yt_data/Orbit.tar.gz.untar/Orbit/orbit_hdf5_chk_00
yt : [INFO     ] 2020-06-26 13:20:52,141 Default to loading None for Orbit dataset
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-344ecbc38ac4> in <module>
----> 1 ds = yt.load_sample("Orbit")

~/repos/yt/yt/utilities/load_sample.py in load_sample(name, specific_file, pbar)
     80         mylog.info("Default to loading %s for %s dataset", file_lookup, name)
     81         loaded_file = os.path.join(base_path, "%s.untar" %fileext,
---> 82                                    name, file_lookup)
     83     else:
     84         mylog.info("Loading %s for %s dataset", specific_file, name)

~/python/anaconda/envs/ytdev37/lib/python3.7/posixpath.py in join(a, *p)
     92                 path += sep + b
     93     except (TypeError, AttributeError, BytesWarning):
---> 94         genericpath._check_arg_types('join', a, *p)
     95         raise
     96     return path

~/python/anaconda/envs/ytdev37/lib/python3.7/genericpath.py in _check_arg_types(funcname, *args)
    151         else:
    152             raise TypeError('%s() argument must be str or bytes, not %r' %
--> 153                             (funcname, s.__class__.__name__)) from None
    154     if hasstr and hasbytes:
    155         raise TypeError("Can't mix strings and bytes in path components") from None

TypeError: join() argument must be str or bytes, not 'NoneType'


```

**Expected outcome**

The dataset should load

**Version Information**
  * Python Version: 3.7.3
  * yt version: Version = 4.0.dev0, Changeset = 9845c5ecd853

**Comment by neutrinoceros** [comment]
Adding the 4.0 label since this feature hasn't shipped yet and it'd be nice to make it more bullet-proof :)

**Comment by munkm** [comment]
This is fixed in yt-project/yt#2951 . If any data formats load incorrectly based on that fix, please file a new issue!!
